src/Components/Blazor/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
```

Comment:
I believe this affects too many projects. @rynowak what subset of the projects in this directory are "client-side Blazor"? (This still isn't clear in #13054)

Reply:
The whole `src/Components/Blazor` subdirectory is exclusively client-side Blazor, so this seems reasonable to me.

I know the naming of this subdirectory is strange. When we ramp up work on shipping client-side Blazor, I expect we'll rename this directory to something like `src/Components/WebAssembly` to avoid confusion.